js/apps/events-page/controllers/eventsPageCtrl.js

```diff
@@ -213,13 +213,18 @@
 		angular.element(document).on('hide.bs.collapse', '.expando-wrapper .collapse', toggleIcon);
 		angular.element(document).on('show.bs.collapse', '.expando-wrapper .collapse', toggleIcon);
 
-		eventsService
-			.get(requestModel)
-			.then(processEvents)
-			.catch(handleFailedEventsGetRequest);
+		if ($location.search().term && $location.search().term.length) {
```

Comment:
would this throw an error if term doesn't exist?

Reply:
No - if `term` doesn't exist, it'll just be undefined and fail the `if`.